src/components/organisms/ModalBase/ModalBase.tsx

```diff
@@ -67,21 +68,26 @@ export interface ModalProps extends ModalFooterProps {
 export interface ModalFooterProps {
   buttonsPosition?: ButtonPositionTypes
   buttons?: ModalButtonProps[]
+  buttonComponent?: ReactElement
 }
 
 const ModalFooter: FC<PropsWithChildren<ModalFooterProps>> = ({
   buttonsPosition = 'right',
   buttons,
+  buttonComponent,
 }) => {
-  if (!buttons) return null
   return (
-    <div className={classes[buttonsPosition]}>
-      {map(buttons, (button, index) => (
-        <Button key={index} {...button}>
-          {button?.children}
-        </Button>
-      ))}
-    </div>
+    <>
+      <div className={classes[buttonsPosition]}>
+        {buttons &&
+          map(buttons, (button, index) => (
+            <Button key={index} {...button}>
+              {button?.children}
+            </Button>
+          ))}
+      </div>
+      {buttonComponent && buttonComponent}
```

Comment:
What does it mean?

Reply:
This is added from FormProgressModal for the buttons to work properly